Description: Bump cabal version number
 Slightly bump the version in the cabal field to 0.2.1.0.1. Otherwise,
 cabal-install will use the dependency information from hackage, trying to
 re-install yesod-json using aeson-native. This dependency was changed in
 0.2.1 to aeson in Debian.
 .
 Also see http://hackage.haskell.org/trac/hackage/ticket/884 for a proper
 solution.
Author: Joachim Breitner <nomeata@debian.org>

--- haskell-yesod-json-0.2.1.orig/yesod-json.cabal
+++ haskell-yesod-json-0.2.1/yesod-json.cabal
@@ -1,5 +1,5 @@
 name:            yesod-json
-version:         0.2.1
+version:         0.2.1.0.1
 license:         BSD3
 license-file:    LICENSE
 author:          Michael Snoyman <michael@snoyman.com>
